modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuilder.java

```diff
@@ -92,7 +92,7 @@ class IndexBuilder implements ManuallyCloseable {
      *
      * <p>Notes:</p>
      * <ul>
-     *     <li>Index is built in batches using {@link BuildIndexReplicaRequest}, which are then transformed into {@link BuildIndexCommand}
+     *     <li>Index is built in batches using {@link BuildIndexReplicaRequest}, which are then transformed into {@link BuildIndexCommandV2}
```

Comment:
I suggest return the old value `BuildIndexCommand`, since essentially nothing has changed.